esx_service/vmdk-opsd

```diff
@@ -112,32 +113,65 @@ stop() {
    if [ -n "${PID}" ]; then
       echo "Stopping $OPSD_TAG with PID=${PID}"
       kill -$SIGTERM ${PID}
+
       local COUNT=0
       while kill -0 ${PID} > /dev/null 2>&1; do
```

Comment:
use ${SIGTERM} instead of 0

Reply:
This is just to check if the process with the given PID is running, no signal is really being sent.